index.html

```diff
@@ -1,14 +1,23 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="utf-8">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.css">
+    <link rel="stylesheet" href="https://unsplash.com/search/photos/bicycle">
     <link rel="stylesheet" href="main.css">
     <title></title>
-  </head>
-  <body>
-    <img src="assignments/images/bike logo clipart.png" alt="">
-    <h1>Baron Crek Ccling</h1>
-    <h2>Bringing you all the best in bicyle gear, equipment, and community since 1999!</h3>
+</head>
+
+<body>
+    <header>
+        <div class="hero">
```

Comment:
I don't think the header is a hero. The hero should likely be the image slider in the middle of the main.